staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -2419,8 +2420,12 @@ func (c *Cloud) waitUntilVolumeAvailable(volumeName KubernetesVolumeID) error {
 		// Unreachable code
 		return err
 	}
-
-	err = wait.Poll(encryptedCheckInterval, encryptedCheckTimeout, func() (done bool, err error) {
+	backoff := wait.Backoff{
+		Duration: volumeCreateInitialDelay,
+		Factor:   volumeCreateBackoffFactor,
+		Steps:    volumeCreateBackoffSteps,
+	}
+	err = wait.ExponentialBackoff(backoff, func() (done bool, err error) {
```

Comment:
> It might even be better to not poll volume immediately after creation at all because volume is never available immediately after creation. But I see that ExponentialBackoff does not offer a way to do that.

Yeah, that could save us a `DescribeVolume` request, which might be helpful for clusters hitting the AWS quota limit.

What if we use `PollUntil` instead? `PollUntil` waits the interval before firing the function.

Reply:
I am fine with adding `time.Sleep(5 * time.Second)` before proceeding with exponential backoff.  It has same effect as anything else and works with exponential backoff..